modules/installation-launching-installer.adoc

```diff
@@ -344,27 +357,29 @@ The AWS access key ID and secret access key are stored in `~/.aws/credentials` i
 .. Select the AWS region to deploy the cluster to.
 .. Select the base domain for the Route 53 service that you configured for your cluster.
 endif::aws[]
-ifdef::azure,ash[]
```

Comment:
Worth noting that I removed the `ash` attribute from this content as I incorrectly introduced it when documenting the ASH installation procedures [1]. ASH does not have a install use case where the installation program is being run without an `install-config.yaml` file. This specific set of content is used for use cases (typically default installation) where an admin runs the installation program without an `install-config.yaml` file.
[1] https://github.com/openshift/openshift-docs/commit/e12ad15ccda17dc6a4970185bc590bf89d8ea2af#diff-86f98220fbab62878a702dcf4ce45ed0c5cd235192d0a79bd889cff480811d2eR244